pages/common/mongoimport.md

```diff
@@ -0,0 +1,24 @@
+# mongoimport
+
+> Imports content from a JSON, CSV, or TSV file into a MongoDB.
+> More information: <https://docs.mongodb.com/database-tools/mongoimport/>.
+
+- Import a JSON file into a specific collection:
+
+`mongoimport --file={{path/to/file}} --uri={{mongodb_uri}} --collection={{collection_name}}`
+
+- Import a CSV file using the first line as the field list into the localhost instance:
```

Comment:
Into the `localhost` instance is really implied in all of the examples that don't have `--uri`; I think we should omit it.
```suggestion
- Import a CSV using the first line of the file to determine property names:
```